halo/evmstaking2/keeper/metrics.go

```diff
@@ -0,0 +1,29 @@
+package keeper
+
+import (
+	"github.com/prometheus/client_golang/prometheus"
+	"github.com/prometheus/client_golang/prometheus/promauto"
+)
+
+var (
+	eventDeliveryHeight = promauto.NewGauge(prometheus.GaugeOpts{
```

Comment:
not sure about this metric. What is the "insight" here? What question is this answering. We already now that, events are processed every 20k blocks, this doesn't add any additional information that we don't already know.

Reply:
Wouldn't it be useful to have a dashboard with the delivery events? I will change this to a counter, which is probably more natural.